doc/changelogs/CHANGELOG_V20.md

```diff
@@ -37,6 +38,168 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="20.3.0"></a>
+
+## 2023-06-08, Version 20.3.0 (Current), @targos
```

Comment:
would have been nice to add your name here @RafaelGSS since you're going to be promoting the release 😊 not a blocker though

Reply:
That's fine. I don't want to spin up all the CI again :P

Reply:
https://github.com/nodejs/node/blob/main/doc/contributing/releases.md?plain=1#L577-L580